content/guides/tutorials/troubleshooting/ts-site-theme-not-loading/index.md

```diff
@@ -6,15 +6,18 @@ dnnversion: 09.02.00
 related-topics: ts-how-to-increase-max-upload-file-size,ts-error-login-ip-filtering-is-currently-disabled,ts-error-another-user-has-taken-action-on-the-page,ts-error-unknown-server-tag-DNNComboBox,ts-error-could-not-load-awssdk,ts-error-sql-timeout,ts-error-argumentnullexception-after-move-upgrade,ts-install-missing-resources,ts-mixed-content-ssl,ts-broken-profile-image,ts-page-remains-in-draft,ts-unable-to-remove-page-redirect-urls,ts-incomplete-content-localization,ts-missing-persona-bar
 ---
 
-# Troubleshooting: Site Theme Does Not Load
+# Troubleshooting: Site Theme Does Not Load or looks weird
```

Comment:
Recommend `# Troubleshooting: Site Theme Does Not Load As Expected`

Reply:
Right, better :-)

Reply:
I confirmed the change